### PR TITLE
Implement PDA 2.20 + Update to VS2022

### PR DESCRIPTION
**Pull request by keikei14**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -4,18 +4,19 @@ on: push
 
 jobs:
   build:
-    runs-on: windows-latest
+    runs-on: windows-2022
```
Comment (BroGamer4256) [inline .github/workflows/build.yml:7]: windows-latest is going to be 2022 and 2022 dosent change MSVC so theres no need to explicitly update, keeping windows-latest should be fine

File: PDA-Loader.sln [hunk 2]
```diff
@@ -33,8 +33,8 @@ Global
 		{69BCF67B-34D9-4424-A2A8-3A056E9D38A6}.Release|x86.Build.0 = Release|Win32
 		{35AA62A4-13DC-403D-85F7-946F9BAE6B01}.Debug|x86.ActiveCfg = Debug|Win32
 		{35AA62A4-13DC-403D-85F7-946F9BAE6B01}.Debug|x86.Build.0 = Debug|Win32
-		{35AA62A4-13DC-403D-85F7-946F9BAE6B01}.Release|x86.ActiveCfg = Release|Win32
-		{35AA62A4-13DC-403D-85F7-946F9BAE6B01}.Release|x86.Build.0 = Release|Win32
+		{35AA62A4-13DC-403D-85F7-946F9BAE6B01}.Release|x86.ActiveCfg = 220|Win32
+		{35AA62A4-13DC-403D-85F7-946F9BAE6B01}.Release|x86.Build.0 = 220|Win32
```
Comment (BroGamer4256) [inline PDA-Loader.sln:37]: Undo this

**Review by BroGamer4256 — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml (hunk 1 above)
Comment (keikei14) [reply .github/workflows/build.yml:7]: Unfortunately, on my last tests keeping it on windows-latest still downloaded VS2019, which errored out on and didnt build correctly.

File: PDA-Loader.sln (hunk 2 above)
Comment (keikei14) [reply PDA-Loader.sln]: Resolved

**Pull request merged**